### DISCUSSION TITLE
Fire Extinguisher Inside Detonator

### DISCUSSION THREAD

**Discussion by Max112358** [FakeFishGames/Barotrauma#15311] [Ideas and suggestions]
I want to be able to place a fire extinguisher inside a detonator. When triggered, it would spray fire extinguishing gas all around itself. Ideally it would drain at the same rate as a person using it, IE not all the extinguisher would be consumed. This way you get more than one use out of it. 

It shouldn't hurt anyone. This is a benign helpful use for an otherwise grief prone item.

This would be great to put near the reactor hooked up to a smoke detector. Or near circuit boxes as a safety. Or near pomegrenade plants.

**Comment by SomeRandomNoobKekeke** [comment]
rather an extinguishing grenade

Mister M already implemented something like that, you could try and see if it works
https://steamcommunity.com/sharedfiles/filedetails/?id=3114087512&searchtext=immersive+repairs
https://youtu.be/m9xCkAU2-WY?feature=shared&t=196

**Comment by Max112358** [comment]
That's good to know, but I would prefer it be in vanilla.
